tests/api_workflow/mocked_api_workflow_client.py

```diff
@@ -110,6 +110,8 @@ def get_tags_by_dataset_id(self, dataset_id, **kwargs):
 class MockedScoresApi(ScoresApi):
     def create_or_update_active_learning_score_by_tag_id(self, body, dataset_id, tag_id, **kwargs) -> \
             CreateEntityResponse:
+        if len(body.scores) > 0 and not isinstance(body.scores[0], float):
```

Comment:
How is this an `AttributeError`? Shouldn't it be a `ValueError` if the scores are not of type float?

Reply:
or a `TypeError`

Reply:
I agree, it is not an `AttributeError`. However, I threw the attribute error to mimic the behaviour of the generated client, which throughs an `AttributeError` if a `float32` is passed, as it tries to access the non-existing `obj.swagger_types` of it.

Reply:
ok